karton/core/base.py

```diff
@@ -54,6 +54,8 @@ def setup_logger(self, level: Optional[Union[str, int]] = None) -> None:
         if not self.identity:
             raise ValueError("Can't setup logger without identity")
 
+        self.log_handler.setFormatter(logging.Formatter("%(message)s"))
```

Comment:
AFAIK we can just use Formatter without arguments
```suggestion
        self.log_handler.setFormatter(logging.Formatter())
```

Ref: https://docs.python.org/3/library/logging.html#logging.Formatter
```
If no fmt is specified, '%(message)s' is used.
```